README.md

```diff
@@ -1,2 +1,9 @@
 # chemnlp
-ChemNLP project
+
+ChemNLP is a project that aims to create an extensive chemistry dataset and use it to train large language models (LLMs) for a wide range of chemistry applications. The resulting models will be capable of predicting properties of materials and molecules, chemical reactions, and even generating computational chemistry code. The potential use cases include accelerating drug discovery, reducing the need for animal experiments, and bringing machine learning to more complex tasks in chemistry. The project builds on prior work that has shown basic chemistry capabilities in LLMs and will use a model setup similar to Med-PaLM and "instruction prompt tuning" (IPT) to fine-tune the LLMs with a few trainable parameters.
+
```

Comment:
## Contributing
ChemNLP is an open-source project - your involvement is warmly welcome! If you're excited to join us, we recommend the following steps:

- 🤔 Looking for ideas? See our [Task Board](https://github.com/orgs/OpenBioML/projects/5/views/1) to see what we may need help with.

- 💡 Have an idea? Create an issue

Feel free to join our [Discord](https://t.co/YMzpevmkiN)!

Reply:
Let's add then the CoC and Contribution Guide later #6